docs/content/direct/multi-wec-aggregated-status.md

```diff
@@ -0,0 +1,191 @@
+
+### Case 2 — Multiple WECs and `wantSingletonReportedState` is false  
+For workloads deployed to multiple WECs, the `.status` field in the WDS remains empty, resulting in no aggregated status propagated.
+
+This leads to limited visibility in ArgoCD, where workloads appear as *OutOfSync* or *Unknown*, even though all WECs are healthy.
+
+**Reference:**  
+- [PR #3297](https://github.com/kubestellar/kubestellar/pull/3297)  
+- [Multi-cluster example in docs](https://docs.kubestellar.io/release-0.28.0/direct/example-scenarios/#scenario-1-multi-cluster-workload-deployment-with-kubectl)
+
+---
+
+## Goal
+
+KubeStellar already collects per-WEC status in `WorkStatus` objects. This enhancement propagates that aggregated information into the `.status` field of the original workload object in the WDS, enabling ArgoCD and similar tools to observe combined readiness and health across all WECs.
+
+Key objectives:  
+- Maintain backward compatibility when `wantMultiWECReportedState` is false, preserving existing behavior.  
+- When `wantMultiWECReportedState` is enabled, provide fallback to the singleton mechanism if only one WEC is bound.  
+- Aggregate per-WEC `WorkStatus` objects and update the `.status` field in WDS workload objects with combined readiness and health status.
+
+---
+
+## Solution Approach
+
+### 1. Extend BindingPolicy CRD
+
+The configuration bit `wantMultiWECReportedState` is added under each `DownsyncModulation` rule (not directly under `BindingPolicy`), consistent with the canonical definition available at [./combined-status/#special-case-for-1-wec](./combined-status/#special-case-for-1-wec).
+
+Example:
+
+```yaml
+apiVersion: policies.kubestellar.io/v1alpha1
+kind: BindingPolicy
+metadata:
+  name: multiwec-nginx
+spec:
+  clusterSelectors:
+  - matchLabels: {"location": "edge"}
+  downsync:
+  - objectSelectors:
+    - matchLabels: {"app.kubernetes.io/name": "nginx-multi"}
+    wantMultiWECReportedState: true
+```
+
+This example follows the [singleton status example](https://docs.kubestellar.io/release-0.28.0/direct/example-scenarios/#scenario-4-singleton-status) but replaces `wantSingletonReportedState` with `wantMultiWECReportedState`.
+
+---
+
+### 2. Introduce `handleMultiWECReportedState()` Function (Planned)
+
+A new controller function, `handleMultiWECReportedState()`, will implement multi-WEC aggregated status reporting using per-kind aggregation logic for common workload kinds such as Deployment, StatefulSet, DaemonSet, ReplicaSet, Job, CronJob, and Pod.
+
+Both `BindingPolicy` and `Binding` inherit the `DownsyncModulation` extension where this flag resides.
+
+---
+
+### 3. Status Aggregation Rules
+
+The aggregation logic depends on the **workload object kind**.
+
+#### Case 1 – Known Workload Kinds
+
+For workload kinds recognized by ArgoCD, the controller applies predetermined field aggregation rules consistent with ArgoCD’s native health evaluation. These include:
+
+- **Deployment**  
```

Comment:
FYI, in real documentation you will want to use real document structure (e.g., `##### Deployments`) here rather than visual formatting.